assets/js/show-task.js

```diff
@@ -32,17 +37,37 @@ function showTask(inputValue, formattedDate, taskList, form, allTasks, id) {
   const DELETE_ICON = icons["DELETE"];
   const deleteButton = createButton("delete", DELETE_ICON, "Delete task");
 
+  const REVERT_ICON = icons["REVERT"];
+  const revertButton = createButton(
+    HIDE_CLASS,
+    REVERT_ICON,
+    "Revert to previous task"
+  );
```

Comment:
inline variable since it's not being reused.

```suggestion
  const revertButton = createButton(
    HIDE_CLASS,
    icons["REVERT"],
    "Revert to previous task"
  );
```